### DISCUSSION TITLE
Seedinfo?

### DISCUSSION THREAD

**Discussion by Chonkblonk** [diasurgical/DevilutionX#7548] [Q&A] [answered]
In multiplayer, I can type /seedinfo in the chat box and then get a little prompt with info, but, uh, how do I get my seed info in singleplayer? 

Something I've been doing for while is that when I come across something in dungeon generation, be it either some kind of error where stuff is broken, or some curious quirk, I get the seedinfo and then take a screenshot, because I remember reading somewhere that noting this down was supposed to be helpful for the devs for some potential future fixing or something (if I understood that right). Playing SP now, I've come across a broken piece, but now I wonder how I'd get the seedinfo here.

**Comment by qndel** [comment, marked as answer]
You don't, because in singleplayer you can provide a save and we can extract all the info from it, in multiplayer current game is not preserved so listing seeds is the only way to get any info about dungeon generated in that game

**Comment by Chonkblonk** [comment]
Aaah, I see.
Here it is then, it should be around on this floor.
[https://files.catbox.moe/ji8if4.hsv](https://files.catbox.moe/ji8if4.hsv)

**Reply by qndel** [reply]
> Aaah, I see. Here it is then, it should be around on this floor. https://files.catbox.moe/ji8if4.hsv

please avoid 3rd party hosting, if you zip it, you can upload directly on github

**Reply by Chonkblonk** [reply]
Ooh. Here.
[single_1.zip](https://github.com/user-attachments/files/17798007/single_1.zip)

**Reply by AJenbo** [reply]
What should be on which floor?

**Reply by Chonkblonk** [reply]
Oh, apologies, a little scatterbrained today.
![bild](https://github.com/user-attachments/assets/35f4e229-13d4-4345-9138-4e41fbbf23e7)

**Reply by AJenbo** [reply]
The catacombs are just full of bugs, I do think I have seen this pattern before but it's good to have a seed that replicates it.
It's one that could conceivable be fixed, feel free to open an issue for it.

**Comment by Chonkblonk** [comment]
![bild](https://github.com/user-attachments/assets/12815952-8f65-4a09-b6d3-89db4be2a28c)
Another from the same seed.
